project/Build.scala

```diff
@@ -118,6 +118,8 @@ object ApplicationBuild extends Build {
       name                := appName + "-social",
       libraryDependencies += "com.typesafe.play" %% "play"       % playVersion % "provided",
       libraryDependencies += "com.typesafe.play" %% "play-ws"    % playVersion % "provided",
+      libraryDependencies += "com.fasterxml.jackson.module" % "jackson-module-scala_2.10" % "2.7.2",
+      libraryDependencies += "net.minidev" % "json-smart" % "1.0.9",

```

Comment:
Why do you introduce jackson and json-smart?

I want to decrease dependent libraries as possible.

Would you use play-json instead of it?